src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -68,6 +69,19 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+
+            // Separately, mount a prerendered server-side Blazor app on /prerendered
+            app.Map("/prerendered", subdirApp =>
+            {
+                subdirApp.UsePathBase("/prerendered");
+                subdirApp.UseStaticFiles();
+                subdirApp.UseRouting();
+                subdirApp.UseEndpoints(endpoints =>
+                {
+                    endpoints.MapFallbackToPage("/PrerenderedHost");
+                    endpoints.MapComponentHub<TestRouter>("app");
+                });
+            });
```

Comment:
This should not be our experience for having multiple entry points as part of an app. But that requires a separate discussion. See https://github.com/aspnet/AspNetCore/issues/8913

Reply:
Sure, this is just what you have to do today, not a claim that it's the ideal dev experience or that we shouldn't change it.